assets/airports/rjbb.json

```diff
@@ -0,0 +1,2732 @@
+		"BERTH": ["N34d09m11.40",	"E134d19m00.00"],
+		"CANDY": ["N33d52m29.80",	"E134d55m41.80"],
+		"CHIZU": ["N35d01m46.20",	"E134d30m25.40"],
+		"DAISY": ["N34d30m02.30",	"E135d03m10.60"],
+		"DANDE": ["N34d13m09.40",	"E135d24m33.90"],
+		"DATIS": ["N33d58m52.00",	"E134d56m13.10"],
+		"EVERT": ["N33d39m25.00",	"E135d07m59.00"],
+		"EVIAN": ["N33d59m42.20",	"E134d36m57.30"],
+		"GATES": ["N34d08m21.40",	"E134d49m02.60"],
+		"GLORY": ["N34d27m36.00",	"E134d44m00.60"],
+		"GOBOH": ["N33d54m36.40",	"E135d06m22.20"],
+		"GUJYO": ["N35d21m50.50",	"E136d34m38.50"],
+		"HABAR": ["N34d19m09.70",	"E133d35m19.60"],
+		"HELEN": ["N34d34m48.90",	"E135d06m51.80"],
+		"HIBIS": ["N34d14m54.80",	"E134d07m30.00"],
+		"JOLLY": ["N34d09m00.60",	"E135d02m25.60"],
+		"JULIA": ["N34d30m08.10",	"E134d58m16.30"],
+		"KAWAT": ["N34d34m19.40",	"E134d33m32.30"],
+		"KEC"  : ["N33d26m51.90",	"E135d47m40.20"],
+		"KRE":   ["N33d32m30.40",	"E133d40m48.60"],
+		"LILAC": ["N34d27m10.70",	"E134d58m42.00"],
+		"MAIKO": ["N34d36m39.70",	"E134d59m49.10"],
+		"MAYAH": ["N34d36m23.70",	"E135d12m11.00"],
+		"NALTO": ["N34d09m00.60",	"E134d43m13.30"],
+		"NARAH": ["N34d29m58.80",	"E135d41m02.90"],
+		"ODORI": ["N34d00m13.20",	"E134d30m11.80"],
+		"SHION": ["N34d25m42.10",	"E134d16m57.40"],
+		"SHTLE": ["N34d49m51.00",	"E136d56m53.80"],
+		"SOUJA": ["N34d37m38.60",	"E133d44m22.50"],
+		"TANTA": ["N34d16m04.90",	"E134d27m29.20"],
+		"TOMOH": ["N34d16m49.60",	"E135d00m20.30"],
+		"TONDA": ["N33d43m43.80",	"E135d22m39.30"],
+		"UENOH": ["N34d50m08.70",	"E136d13m33.70"],
+		"WASYU": ["N34d28m17.50",	"E133d23m01.10"],
+		"WENDY": ["N34d32m09.50",	"E134d50m30.80"],
+		"YAGYU": ["N34d32m39.30",	"E135d57m38.10"],
+		"YME"  : ["N35d28m50.50",	"E135d08m13.30"],
+		"YOSAK": ["N33d47m04.70",	"E134d06m38.40"],
+		"HAMAR": ["N34d24m34.70",	"E135d03m04.10"],
+		"KCC"  : ["N35d15m55.00",	"E136d54m53.70"],
+		"KCE"  : ["N34d37m52.00",	"E135d13m42.00"],
+		"KNE"  : ["N34d25m48.00",	"E135d15m06.00"],
+		"KTE"  : ["N34d12m45.00",	"E134d01m21.00"],
+		"KYOTO": ["N34d49m49.70",	"E135d44m48.80"],
+		"LINDA": ["N34d30m15.80",	"E134d51m40.10"],
+		"NANKO": ["N34d36m38.70",	"E135d21m21.00"],
+		"OGURA": ["N34d47m58.70",	"E135d34m24.90"],
+		"OYE"  : ["N34d45m01.00",	"E133d50m06.00"],
+		"SIGAK": ["N34d53m07.70",	"E136d02m52.70"],
+		"TSC"  : ["N34d07m47.00",	"E134d36m31.00"],
+		"UMAJI": ["N33d45m00.50",	"E134d09m00.60"],
+		"BRAVE": ["N33d25m50.00",	"E133d33m30.00"],
+		"FUE":   ["N32d40m07.00",	"E128d49m36.00"],
+		"KARIN": ["N33d40m40.00",	"E134d11m50.00"],
+		"KISEI": ["N33d28m04.00",	"E135d19m44.00"],
+		"OOITA": ["N33d13m13.00",	"E131d42m12.00"],
+		"RANDY": ["N34d06m35.00",	"E133d53m51.00"],
+		"SALTY": ["N33d51m10.00",	"E132d55m31.00"],
+		"STORK": ["N32d56m48.00",	"E133d14m35.00"],
+		"SUC":   ["N32d45m21.00",	"E132d59m48.00"],
+		"TAPOP": ["N32d40m01.00",	"E136d07m57.00"]
+	},
+    "runways": [{
+        "name": ["06L", "24R"],
+        "name_offset": [
+            [0, 0], [0, 0]
+        ],
```

Comment:
If no offset is being applied, should we omit this? It seems like it's often unnecessary to offset them anyway, as the numbers are an aid, and not a necessity.